CHANGELOG.md

```diff
@@ -56,3 +56,4 @@
 * [BUGFIX] Modules: fix a module waiting for other modules depending on it before stopping. #141
 * [BUGFIX] Multi KV: fix panic when using function to modify primary KV store in runtime. #153
 * [BUGFIX] Lifecycler: if the ring backend storage is reset, the instance adds itself back to the ring with an updated registration timestamp set to current time. #165
+* [BUGFIX] Ring: fix bug where ingesters may appear unhealthy even though they are not #172
```

Comment:
This bug was reported for ingesters, but could potentially affect any hash ring, so I would generalise it. Moreover, dskit is a general purpose library and it has no notion of "ingesters" (unfortunately, due to some legacy code there are still references to it in dskit, but that's something we should cleanup).

```suggestion
* [BUGFIX] Ring: fix bug where hash ring instances may appear unhealthy in the web UI even though they are not. #172
```